doc/env.md

```diff
@@ -0,0 +1,65 @@
+**WORK IN PROGRESS, NOT YET COMPLETE**
+
+# Env
+
+The opaque data structure containing the environment in which the request is being run.
+
+The Env object is usually created and passed by the calling system.
```

Comment:
Would it be better to say 

"is usually created and passed by the Node.js runtime or node-addon-api infrastructure"

The same comment also applies other places were we say "calling system"